specs/governance/alligator-interop.md

```diff
@@ -0,0 +1,136 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+  - [Interface](#interface)
+    - [Core Functions](#core-functions)
+  - [Diagram](#diagram)
+  - [Implementation](#implementation)
+  - [Backwards Compatibility](#backwards-compatibility)
+  - [Security Considerations](#security-considerations)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Overview
+
+The goal of this specification is to allow interoperability between the base and superchains for that use the [`Alligator`](alligator.md) predeploy. This allows the support of the
```

Comment:
One common misconception is that each chain is called a superchain, but really the superchain is made up of individual chains. What do you mean the base here?

Reply:
Explicated the use of other OP chains that are not mainnet and their interaction with the `Alligator` contract